### ISSUE TITLE
Scroll doesn't work in any popup windows

### ISSUE THREAD

**Issue by solonifer** [darktable-org/darktable#14278]
### Describe the bug

1. Scroll doesn't work in any popup windows (example in attached image)
<img width="224" alt="Screenshot 2023-04-21 at 4 59 08" src="https://user-images.githubusercontent.com/608227/233523376-3431d2f6-2df0-4d28-861a-58d6692deeeb.png">
<img width="607" alt="Screenshot 2023-04-21 at 4 52 15" src="https://user-images.githubusercontent.com/608227/233523378-0a331c5c-59bd-48d7-9227-947a1c6176d8.png">

2. darkroom (tab at the top) -> technical (section) -> modules list (not scrollable)

### Where did you install darktable from?

darktable.org

### darktable version

4.2.1

### What OS are you using?

Mac

### What is the version of your OS?

Ventura 13.0 (22A380)

### Describe your system?

MacBook Pro M1

### Are you using OpenCL GPU in darktable?

I dont know

**Comment by ralfbrown** [comment]
There's a known bug with Ventura and mouse scrolls, but I thought 4.2.1 had fixed it.  If you use arrow keys first, can you then use mouse scrolling to change selection in the popup list?  If so, it's the same bug.

**Comment by zisoft** [comment]
Works for me with current master on Ventura 13.3.1, Intel.

@MStraeten: Can you verify on your M1 system?

**Comment by MStraeten** [comment]
there's something weird even with current master - need to investigate reproducible scenarios. seems to depend on screen resolution...
just found such behaviour on my external display (even worse: just displayed one entry for preferences - workflow selection on external display but all 4 on notebook display) 
![image](https://user-images.githubusercontent.com/39386816/233681648-6d2805d6-18b9-48b3-856d-69d531d49808.png)
![image](https://user-images.githubusercontent.com/39386816/233681882-17a72c90-3839-4fc4-bab0-50d79f4b4928.png)
on notebook display: 
![image](https://user-images.githubusercontent.com/39386816/233682169-1d64791f-72af-49a5-be7d-6d7faab8d967.png)

**Comment by MStraeten** [comment]
btw there seems to be more issues if using external and internal display - will file a further issue if i can exclude randomness

**Comment by zisoft** [comment]
This is how it looks on my MacBook Pro, using the scroll wheel:

https://user-images.githubusercontent.com/4083281/233690860-bfef527b-292f-4312-b93a-8c868fb8e74c.mov

**Comment by MStraeten** [comment]
![image](https://user-images.githubusercontent.com/39386816/233695055-904d938d-d31e-4de5-b3cf-6f45ab0b02ed.png)

even darktable should be opened on the external display according to the window settings in darktablerc the initialization is done on the internal display  - maybe if display properties are defined there this can mess up the behaviour on the external display

so @solonifer: are you using a 2 display configuration?

**Comment by dterrahe** [comment]
So your _external_ display gets cut off, even though the _internal_ one has a higher resolution? Strange. How are they laid out relative to each other?

The menus you are showing are standard gtk ones (not the homemade dt combos) so this seems like entirely an upstream bug.

**Comment by MStraeten** [comment]
window parameters from darktablerc if window is maximized:
internal:
```
ui_last/window_h=1051
ui_last/window_w=1728
ui_last/window_x=0
ui_last/window_y=38
```
external:
```
ui_last/window_h=1387
ui_last/window_w=2560
ui_last/window_x=1728
ui_last/window_y=25
```

**Comment by dterrahe** [comment]
Ah, so your internal retina display halves the resolution. Think I read about that a long time ago; that MacOS does some scaling to support old apps.

Does it make a difference _where_ (how high) the menu appears in how much of it gets cut off. I would "expect" the external display to get cut off at 1051/1387th from the top. Or something. Still Gtk bug, no? They should have come across it?

**Comment by MStraeten** [comment]
haven't found related gtk issues in their gitlab yet...
but screen position matters:

https://user-images.githubusercontent.com/39386816/233736690-fe3af3bb-e347-493b-80e0-a90d46b7824f.mp4

**Comment by MStraeten** [comment]
a build using gtk3 3.24.38 doesn't show this behaviour - but has redrawing issues. Since the latter is just annoying and can be handled by additional scaling attempts (if it occurs) using gtk3 3.24.38 seems to be the better solution.
so here a 4.4.2 arm release build with gtk3 3.24.38
[darktable-4.4.2_arm64_gtk3.24.38.dmg](https://mega.nz/file/bOwnEaRK#VXAk3zXgcOjI2YOTNeppgMwddH9P5i5twjiXbz1gPqM)
(not codesigned, so you might need to allow execution via terminal: " xattr -c <path/to/application.app> "
please confirm if that also solves your issue ...

**Comment by MStraeten** [comment]
a remaining issue with improper rendering of Roboto font at 72dpi was fixed with #15093
So this can be closed
